### DISCUSSION TITLE
Unescaped special characters break template

### DISCUSSION THREAD

**Discussion by jtinsky** [sheltermanager/asm3#1579] [Q&A / Help] [answered]
In the default animalview template the contents of the following variable `<meta property="og:description" content="$$ANIMALCOMMENTS$$" />` aren't escaped so the comment text which contains quotation marks closes the content attribute resulting in what's shown in the image below.  Is there a filter or some other method of escaping those characters in the templates? I can remove the quotes for this entry but we have volunteers entering comments so it's likely to happen again at some point.

![gladys](https://github.com/user-attachments/assets/41d2cc29-c0c3-4ac7-bd54-f28013d61934)

**Comment by bobintetley** [comment, marked as answer]
Yes, use ```$$DESCRIPTIONATTR$$``` instead, it escapes everything so it can be used in an HTML attribute.

**Reply by jtinsky** [reply]
And the winner for quickest reply goes to... Thank you sir!
